projects/emissary-ingress/emissary/docker/linux/emissary-image/Dockerfile

```diff
@@ -0,0 +1,65 @@
+ARG BASE_IMAGE
+
+FROM public.ecr.aws/k1e6s8o8/eks-distro-minimal-base-python-builder:3.9-latest.2 as emissary-builder
+
+ENV PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/go/bin:/buildroot/bin
+
+RUN install_rpm libcap htop && \
+    install_binary /usr/bin/curl /usr/bin/chgrp /usr/bin/chmod && \
+    cleanup "emissary"
+
+WORKDIR /buildroot
+
+ADD emissary/python python
+
+RUN time pip3 install --no-deps -e python
```

Comment:
All of these runs, most likely, are going to need to be combined into as few layers as possible or will run out of space on presubmit.

Reply:
Resolved in https://github.com/aws/eks-anywhere-build-tooling/commit/22874139be342205874c4a731894f9ad27c3e657